src/platform/web/config.ts

```diff
@@ -79,6 +81,11 @@ const Config: IPlatformConfig = {
     return byteArray.buffer;
   },
   isWebworker: isWebWorkerContext(),
+  push: {
+    platform: DevicePlatform.Browser,
+    formFactor: DeviceFormFactor.Desktop,
```

Comment:
Yeah it's intentional, the `platform` will always be browser since this is the config for the web platform (however will be different for react native). I looked at detecting form factor for web clients and decided that there wasn't a reliable + portable way to do it (and it's of questionable value anyway) so decided to just keep it as desktop for simplicity.

Reply:
Do we even use `formFactor` somewhere? Is this part of the realtime implementation and it's a minor metadata for some analytics?

Reply:
Copying Paddy's reply from https://github.com/ably/ably-js/pull/1775#discussion_r1647220328:

> >Do we even use formFactor somewhere? Is this part of the realtime implementation and it's a minor metadata for some analytics?
> No, it's not used anywhere - it's just informative for anyone who looks at the registration, eg via the push admin API